### DISCUSSION TITLE
Add support for PowerSeries Neo

### DISCUSSION THREAD

**Discussion by ghost** [taligentx/dscKeybusInterface#111] [dscKeybusInterface Discussion]
hi, i received today Dsc PowerSeries Neo (https://www.dsc.com/dsc-product-families/Neo/1), and tested with current firmware, as expected, it's not working...
question, it,s possible to add support for this model?
i can send to firmware developers, dsc panel to reverse new protocol. Best regards
dsc neo screens:
![IMG_20190722_135722](https://user-images.githubusercontent.com/20506433/61633404-b0565d80-ac97-11e9-88e1-15b0da188f43.jpg)
![IMG_20190722_135726](https://user-images.githubusercontent.com/20506433/61633405-b0565d80-ac97-11e9-9520-640b9833d908.jpg)
![IMG_20190722_135743](https://user-images.githubusercontent.com/20506433/61633406-b0565d80-ac97-11e9-95f6-854d1b855451.jpg)
![IMG_20190722_140300](https://user-images.githubusercontent.com/20506433/61633410-b0eef400-ac97-11e9-85f0-dd406e23e404.jpg)

**Comment by taligentx** [comment]
Hi @connected201 - please refer to the `README.md`, this has already been addressed.  Neo uses a totally different encrypted protocol so unless someone is able to crack the encryption these systems cannot be directly interfaced.

**Comment by PascalSI** [comment]
Hi @connected201 , where can i see how you made a printed circuit board for such a case?

**Comment by ghost** [comment]
hi @PascalSI , i wil made some changes, and add gsm support , and i will public all sources

**Comment by ArduinoQ97** [comment]
@connected201 Hi.
Just wondering whether you manage to find a way to use connect to Neo series?
@taligentx Do you have any update on Neo series or any advice on how can to connect with them?
Thanks

**Comment by taligentx** [comment]
Hi @ArduinoQ97 - no news on the Neo series, the protocol would need to be decrypted by someone.

**Comment by fariddergal** [comment]
Hi all , any news about neo interface? Thanks

**Reply by taligentx** [reply]
Hi @fariddergal - no news on the Neo series, the protocol would need to be decrypted by someone.

**Reply by fariddergal** [reply]
There is a communication board tl280e I have read that it comunícate a with the main board through tp Link, I have read in the specification that it can be configured to un able the encryption locally , the ñoint is to see if any developer is putting his eyes on this topic otherwise will be very hard

**Reply by fariddergal** [reply]
> There is a communication board tl280e I have read that it comunícate a with the main board through PC Link, I have read in the specification that it can be configured to un able the encryption locally , the ñoint is to see if any developer is putting his eyes on this topic otherwise will be very hard
